fetch.bs

```diff
@@ -8246,6 +8250,20 @@ of running <a for=Body>consume body</a> with <a>this</a> and <a>parse JSON from
 of running <a for=Body>consume body</a> with <a>this</a> and <a>UTF-8 decode</a>.
 </div>
 
+<div algorithm>
+<p>The <dfn method for=Body><code>textStream()</code></dfn> method steps are:</p>
+
+<ol>
+ <li><p>If <span>this</span>'s <a for=Body>body</a> is null, throw a {{TypeError}}.
+
+ <li><p>Let <var>stream</var> be <a>this</a>'s <a for=Body>body</a>'s <a for=body>stream</a>.
+
+ <li><p>Let <var>decoder</var> be a new {{TextDecoderStream}} object.
```

Comment:
This and the next step would need improving if there's interest in this method.

The constructor steps for `TextDecoderStream` aren't implicitly run by just saying "new object", but what I'm after is an UTF-8 decoder with the default settings.

Reply:
We should probably expose an algorithm to create a `TextDecoderStream` from other specifications, [like we do for Streams](https://streams.spec.whatwg.org/#other-specs).